### DISCUSSION TITLE
Computing Conditional Probabilities

### DISCUSSION THREAD

**Discussion by ryanr0401** [CamDavidsonPilon/lifelines#1442] [Q&A]
Hi, 
I am currently working on predicting customer survival for a gas supplier. They require that I compute the conditional probabilities for survival, e.g given I made it to month 3 whats the prob of survival to month 4,5 and 6 ect. I managed to compute these using Kaplan-Meier however they said they would like the inclusion of covariates in the model. I then thought cox-ph model would be best. However I was using the method: for i in range of my timeline,  kmf.survival_function / kmf.predict(i) to compute these probabilities. However, using cox I have noticed I get a predicted survival function for each entry. Has anyone a solution to this problem or any ideas on how to solve it

**Comment by CamDavidsonPilon** [comment]
Hi there, have you tried using `predict_survival_function` with the kwarg `conditional_after`? https://lifelines.readthedocs.io/en/latest/Survival%20Regression.html#prediction-on-censored-subjects
